ui/src/components/EmptyCard.tsx

```diff
@@ -30,7 +30,7 @@ export default function EmptyCard({
         <div className="max-w-[90%] space-y-1.5 text-center md:max-w-[60%]">
           <div className="space-y-2">
             {IconElm && (
-              <IconElm className="mx-auto h-6 w-6 text-blue-600 dark:text-blue-400" />
+              <IconElm className="mx-auto h-5 w-5 text-blue-600 dark:text-blue-600" />
```

Comment:
Just something I noticed working on the Tailwind upgrade... the `h-5 w-5` can be written `size-5` if you prefer :)

Reply:
Yeah, I saw that too. Let's use that moving forward or when we see it.